account_tax_rounding_method/models/account_tax.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    @api.model
+    def _get_tax_rounding_method(
+        self,
+        company,
+        partners=None,
```

Comment:
We now expect no cases of getting a recordset.

```suggestion
        partner=None,
```